sdmetrics/single_table/privacy/loss.py

```diff
@@ -33,14 +35,13 @@ class InverseCDFDistance(LossFunction):
     This loss function first applies the fitted cdfs to every single entry (i.e. turning
     the numerical values into their respective percentiles) and then measures the Lp distance
     to the pth power, between the predicted value and the real value.
+
```

Comment:
Is it our own convention to put the `args` in the class description instead of `__init__`?